ods_ci/libs/DataSciencePipelinesAPI.py

```diff
@@ -294,27 +299,37 @@ def retrieve_auth_url(self):
         response = response[host_begin_index:]
         host = response[: response.index(":")]
         # host[4:] means that we need to remove '.api' from the url
-        return f"https://oauth-openshift.apps.{host[4:]}/oauth/authorize?response_type=token&client_id=openshift-challenging-client"
+        return (
+            f"https://oauth-openshift.apps.{host[4:]}/oauth/authorize?response_type=token&client_id=openshift"
+            f"-challenging-client"
+        )
 
     def get_default_storage(self):
         result, _ = self.run_oc("oc get storageclass -A -o json")
         result = json.loads(result)
-        for storage_class in result['items']:
-            if storage_class['metadata']['annotations']['storageclass.kubernetes.io/is-default-class'] == 'true':
+        storage_class_name = ""
+        for storage_class in result["items"]:
+            if (
+                storage_class["metadata"]["annotations"][
+                    "storageclass.kubernetes.io/is-default-class"
+                ]
+                == "true"
+            ):
+                storage_class_name = storage_class["metadata"]["name"]
                 break
-        return storage_class['metadata']['name']
+        return storage_class_name
```

Comment:
was the linter complaining about this?

Reply:
IIRC the reason for this was that the method returned data under some condition, but when the loop finish there were no return value. Another possible solution would be to return null after for loop.